# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fourier/install-site.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fourier-0.3.0/Makefile.vanilla	2007-11-30 15:58:00.000000000 +0100
+++ fourier-0.3.0/Makefile	2007-11-30 15:58:23.000000000 +0100
@@ -1,7 +1,7 @@
 #  Use gimptool-2.0 to sett these variables
 GIMPTOOL=gimptool
 PLUGIN_BUILD=$(GIMPTOOL) --build
-PLUGIN_INSTALL=$(GIMPTOOL) --install-bin
+PLUGIN_INSTALL=$(GIMPTOOL) --install-admin-bin
 GCC=g++
 LIBS=$(shell pkg-config fftw3 --libs)
 CFLAGS=-O2 $(shell pkg-config fftw3 --cflags)
